source/Calamari.Terraform/DestroyTerraformConvention.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using Calamari.Deployment;
+using Calamari.Integration.FileSystem;
+using Calamari.Integration.Substitutions;
+
+namespace Calamari.Terraform
+{
+    public class DestroyTerraformConvention : TerraformConvention
+    {
+        public DestroyTerraformConvention(ICalamariFileSystem fileSystem, IFileSubstituter fileSubstituter) : base(fileSystem, fileSubstituter)
+        {
+        }
+
+        protected override void Execute(RunningDeployment deployment, Dictionary<string, string> environmentVariables)
+        {
+            using (var cli = new TerraformCLIExecutor(fileSystem, deployment, environmentVariables))
+            {
+                cli.ExecuteCommand("destroy", "-force", "-no-color", cli.TerraformVariableFiles,
```

Comment:
The other two commands we take the return type and verify success. Do we not need to do that for `Destroy`?

Reply:
We are, the `ExecuteCommand` has a bunch of overloads, and this one verifies success